ui/page/components/components.go

```diff
@@ -714,3 +733,122 @@ func CoinImageBySymbol(icons *load.Icons, coinName string) *decredmaterial.Image
 	}
 	return nil
 }
+
+func WalletLiveTickets(w *dcrlibwallet.Wallet) ([]dcrlibwallet.Transaction, error) {
```

Comment:
This seems like a candidate for a dcrlibwallet `wallet.UnspentTickets` method. Unspent, not Live because unmined tickets are not live but I see they are returned by this function as well.

Reply:
can you please link the method, can't find it

Reply:
Which method? I mean to move this `WalletLiveTickets` method to dcrlibwallet and call it 
~`wallet.UnspentTickets`~ `wallet.UnspentUnexpiredTickets` which better represents the kind of tickets returned by the method.